lib/ecto/schema.ex

```diff
@@ -466,7 +466,7 @@ defmodule Ecto.Schema do
       Module.register_attribute(__MODULE__, :ecto_raw, accumulate: true)
       Module.register_attribute(__MODULE__, :ecto_autogenerate, accumulate: true)
       Module.register_attribute(__MODULE__, :ecto_autoupdate, accumulate: true)
-      Module.register_attribute(__MODULE__, :ecto_redacted_fields, accumulate: true)
+      Module.register_attribute(__MODULE__, :ecto_redact_fields, accumulate: true)
       Module.put_attribute(__MODULE__, :ecto_derive_inspect_for_redacted_fields, true)
```

Comment:
I kept redacted here because I felt it reads better. I was also on the fence about `__schema__(:redact_fields)` vs `__schema__(:redacted_fields)` but I think both are good?